lms/static/js/spec/search/search_spec.js

```diff
@@ -655,6 +655,7 @@ define([
 
             function navigatesToSearch () {
                 var requests = AjaxHelpers.requests(this);
+                Backbone.history.start();
```

Comment:
This is needed because `loadUrl` is changed in new Backbone version and tries to access `this.root` which is only initialized in `Backbone.history.start`